src/CMakeLists.txt

```diff
@@ -24,21 +17,24 @@ if (${PROJECT_NAME}_STATIC_LIB)
 else()
   set(CMAKE_MACOSX_RPATH 1)
   add_library(${PROJECT_NAME} SHARED ${library_sources})
-  if(UNIX)
-    set_target_properties(tess2 PROPERTIES COMPILE_FLAGS "-fPIC")
-  endif()
 endif()
 
+target_include_directories(${PROJECT_NAME} PRIVATE
+  ${PROJECT_SOURCE_DIR}/include
+  ${PROJECT_SOURCE_DIR}/src
```

Comment:
Shouldn't these be public?

Reply:
Done, These directories were changed to `PUBLIC`.